### ISSUE TITLE
ui-boostrap 1.1.2 and xeditable 0.1.9 Datepicker not popup

### ISSUE THREAD

**Issue by msenthu** [vitalets/angular-xeditable#428]
All,

I am using angular 1.4.8, ui-boostrap 1.1.2 and xeditable 0.1.9, The Datepicker is not popup. Can you help me resolve this issue?

Thanks,
Senthil

**Comment by ckosloski** [comment]
The ui-bootstrap tags need to be updated to use "uib".  See pull request #406

**Comment by msenthu** [comment]
I am new to the angularjs, My code is simillar to this. Could you please let me know where do I need to change?

<div ng-controller="BsdateCtrl">
  <a href="#" editable-bsdate="user.dob" e-datepicker-popup="dd-MMMM-yyyy">
    {{ (user.dob | date:"dd/MM/yyyy") || 'empty' }}
  </a>
</div>

**Comment by msenthu** [comment]
Thanks for the solution that worked, but still I am getting lot of console errors like below

> TypeError: ngModelCtrl.$setValidity is not a function
> 
> Error: [$compile:ctreq] Controller 'ngModel', required by directive 'uibDatepicker', can't be found!

But I do have ng-Model/ e-ng-model that didnt help resolve the error. Also it is not using the format I specified. Any idea?

`e-uib-datepicker-popup="yyyy-MMM-dd"`

**Comment by ckosloski** [comment]
Your definition should be:
`data-e-datepicker-popup="yyyy-MMM-dd"`

**Comment by msenthu** [comment]
Thanks for the quick reply,  Here is my code i tried different options but both didn't work

```
<a href="#"  ng-model="lastReviewed" editable-bsdate="lastReviewed" data-e-datepicker-popup="yyyy-MMM-dd"
       e-ng-click="opened = true" e-is-open="opened"       >
        {{ (lastReviewed | date: "yyyy-MMM-dd") || 'Not reviewed' }}
</a>
```
Above code I am not getting the datapicker popup

and
```
<a href="#"  ng-model="lastReviewed" editable-bsdate="lastReviewed" data-e-uib-datepicker-popup="yyyy-MMM-dd" data-e-datepicker-popup="yyyy-MMM-dd"
       e-ng-click="opened = true" e-is-open="opened"       >
        {{ (lastReviewed | date: "yyyy-MMM-dd") || 'Not reviewed' }}
</a>
```
in the above code I get the datepicker popup but format is different (screen shot below). I would like the date appear as yyyy-MMM-dd format

![image](https://cloud.githubusercontent.com/assets/17090660/13407985/3dee6a9a-def9-11e5-91c5-9caef016faf2.png)

**Comment by ckosloski** [comment]
You need to remove:
`data-e-uib-datepicker-popup="yyyy-MMM-dd"`

You should only have:
`data-e-datepicker-popup="yyyy-MMM-dd"`

**Comment by msenthu** [comment]
If i downgrade my angularjs-xeditable to 0.1.8 then it works as expected. It is not working on 0.1.9 and 0.1.10.

Thanks

**Comment by eugef** [comment]
Fixed by #443
